notebooks/tutorial_resampling_data.ipynb

```diff
@@ -17,17 +17,17 @@
     {
```

Comment:
<p>Resampling now returns the data in the order of (space, time), irrespective of the original order (tested the code below and it still outputs (space, time) once you select the variable.</p><p> </p><p>Is there a reason for this order? Otherwise I'm in favor of sticking to what I know to be 'common practice'. Many netcdfs (cmips and e.g. ERA5 data) are stored with order (time, space). </p><p> </p><pre class="ql-syntax" spellcheck="false">import xarray as xr

time_index = pd.date_range('20171020', '20211001', freq='15d')

np.random.seed(0)
temperature = 15 + 8 * np.random.randn(len(time_index), 2, 2)
precipitation = 10 * np.random.rand(len(time_index), 2, 2)

lon = [[-99.83, -99.32], [-99.79, -99.23]]
lat = [[42.25, 42.21], [42.63, 42.59]]


ds = xr.Dataset(
    data_vars=dict(
        temperature=(["time", "x", "y"], temperature),
        precipitation=(["time", "x", "y"], precipitation),
    ),
    coords=dict(
        time=time_index,
        lon=(["x", "y"], lon),
        lat=(["x", "y"], lat)
    ),
    attrs=dict(description="Weather related data."),
)
ds
</pre><p> </p><p> </p><pre class="ql-syntax" spellcheck="false">ds_r = s2spy.time.resample(calendar, ds)
ds_r.temperature
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/s2spy/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>